packages/ses/test/lockdown.test.js

```diff
@@ -25,4 +35,25 @@ test('lockdown returns boolean or throws in SES', t => {
     () => lockdown({ noTameRegExp: true }),
     'throws when attempting to untame RegExp',
   );
+
+  // Compartment constructor does not throw after lockdown.
+  (() => {
+    const c = new Compartment({}, {}, {});
+    const cf = c.evaluate('Function.prototype.constructor');
+    t.notEqual(
+      cf,
+      Function,
+      'after lockdown, function constructor must be tamed inside compartments',
+    );
+
+    const d = new Compartment({}, {}, {});
+    const df = d.evaluate('Function.prototype.constructor');
+    t.equal(
+      cf,
+      df,
+      'after lockdown, every compartment must have the same tamed function constructor',
```

Comment:
The phrase "function constructor" can validly be taken to mean the function that one does `new` on to create functions. I think all relevant error messages should spell out `Function.prototype.constructor` or in spec-ease `%FunctionPrototype%.constructor`